src/FakeCLI.java

```diff
@@ -0,0 +1,2 @@
+public class FakeCLI {
+}
```

Comment:
I totally thought I'd separated these into two different PR's so I'm not sure how I messed it up.